libraries/integration.rb

```diff
@@ -90,7 +90,7 @@ def config_file_content
       end
 
       # Fetch the remote executable tarball if the install method is set to `tarball`
-      poise_archive new_resource.remote_url do
+      archive_file new_resource.remote_url do
```

Comment:
`archive_file` is available from 15+ https://docs.chef.io/resources/archive_file/
And we provide support for 12.15+ as per: https://github.com/newrelic/infrastructure-agent-chef#chef